toolchain/check/check_fuzzer.cpp

```diff
@@ -10,6 +11,18 @@
 
 namespace Carbon::Testing {
 
+static InstallPaths install_paths;
```

Comment:
Is this fully conformant with static init requirements? I'm surprised you're not structuring this more as a pointer or similar.

Reply:
No, the pointer is a better approach, thanks for suggesting. I've switched to that.